src/Symfony/Bridge/Monolog/Tests/Handler/ConsoleHandlerTest.php

```diff
@@ -110,31 +110,18 @@ public function testGetFormatter()
 
     public function testWritingAndFormatting()
     {
-        $output = $this->getMock('Symfony\Component\Console\Output\ConsoleOutputInterface');
+        $output = $this->getMock('Symfony\Component\Console\Output\OutputInterface');
         $output
             ->expects($this->any())
             ->method('getVerbosity')
             ->will($this->returnValue(OutputInterface::VERBOSITY_DEBUG))
         ;
         $output
-            ->expects($this->once())
+            ->expects($this->any())
```

Comment:
Ah that was trying to achieve something else and then I gave up and forgot to change it back. Let me change it back quickly.